apps/minaintyg/src/pages/Certificate/CertificatePage.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { faker } from '@frontend/fake'
```

Comment:
I think we should have a talk within the team about how to write tests in frontend. Right now it feels like snapshot tests are the main technique for testing. My opinion is that snapshot tests can be used when necessary but that they shouldn't replace unit tests.